src/models/Settings.js

```diff
@@ -135,8 +135,9 @@ function (Okta, Q, Errors, BrowserFeatures, Util, Logger, OAuth2Util, config) {
       'customButtons': ['array', false, []],
 
       //Registration
-      'registration': 'object',
-      'registration.click': 'function'
+      'registration.enable': ['boolean', false, false],
```

Comment:
That was a suggestion from Reman. If the developer doesn't override anything, it will be the default behavior.
So instead to turn on a feature by empty json. It will be more explicitly to say `enable: true`